generators/pkg-cypress/templates/config.yml

```diff
@@ -0,0 +1,40 @@
+jobs:
+  e2e-tests:
+    working_directory: ~/project
+    docker:
+      - image: 'cypress/base:14.18.1'
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - restore_cache:
+          keys:
+            - v2-deps-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - v2-deps-{{ .Branch }}-
+            - v2-deps-
+      - run: npm ci
+      - save_cache:
+          key: v2-deps-{{ .Branch }}-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/.npm
+            - ~/.cache
```

Comment:
```suggestion
```
questi possiamo toglierli giusto?

Reply:
@CristinaMuntianu ok quindi non riusciamo a cachare durante il primo job anche la cache di cypress? Invece di reinstallare tutto al job dell'e2e?

Reply:
@antoniogiordano credo di si, ma devo fare delle prove.. posso fare dei commit su un mio vecchio task di Moneytorize per provare?

Reply:
@CristinaMuntianu sì certo, riparti da master, fai un pull, crea in locale un branch tipo cristinamuntianu/cypress-ci-tests e fai commit, push e PR